commercetools-models/src/main/java/io/sphere/sdk/carts/CartOrigin.java

```diff
@@ -0,0 +1,30 @@
+package io.sphere.sdk.carts;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.sphere.sdk.models.SphereEnumeration;
+
+/**
+ * An enumeration that allows to track cart's origin.
+ */
+public enum CartOrigin implements SphereEnumeration{
+
+
+    /**
+     * The cart was created by the customer. This is the default value.
+     */
+    CUSTOMER,
+    /**
+     * The cart was created on behalf of the customer by a customer representative in the Merchant Center or a similar tool of the Merchant.
+     */
+    MERCHANT;
+
+
+    public static CartOrigin defaultValue() {
```

Comment:
This is a nice idea 👍  We should add it to all of our enums or at least add it later to our RAML enum code generator :wink: